src/js/components/sharedComponents/header/NavBar.jsx

```diff
@@ -1,72 +1,90 @@
-                    <span className="logo-sr">USAspending.gov</span>
-                </a>
-            </div>
-        </div>
-        <div className="mobile-menu">
-            <button className="usa-menu-btn">
-                <span className="nav-lines" />
-            </button>
-        </div>
-        <nav role="navigation">
-            <div className="menu-container">
-                <div className="upper-menu">
-                    <ul className="small-menu">
-                        <li>
-                            <a
-                                href="/#/about"
-                                rel="noopener noreferrer"
-                                title="About">
-                                About
-                            </a>
-                        </li>
-                        <li>
-                            <a
-                                href="https://usaspending-help.zendesk.com/hc/en-us"
-                                target="_blank"
-                                rel="noopener noreferrer"
-                                title="Help">
-                                Help
-                            </a>
-                        </li>
-                        <li>
-                            <GlossaryButtonWrapperContainer child={NavBarGlossaryLink} />
-                        </li>
-                    </ul>
+const propTypes = {
+    homepage: PropTypes.bool
+};
+
+const NavBar = (props) => {
+    let homepageClass = '';
+    if (props.homepage) {
+        homepageClass = 'homepage';
+    }
+
+    return (
+        <nav
+            className="nav-container"
+            role="navigation">
+            <div className="logo">
+                <div className={`usa-logo ${homepageClass}`} id="logo">
+                    <a href="#/" title="USAspending.gov Home" aria-label="USAspending.gov Home">
+                        <span className="logo-sr">USAspending.gov</span>
+                    </a>
                 </div>
-                <div className="lower-menu">
-                    <ul className="nav-menu">
-                        <li className="profile-item">
-                            <ProfileButton />
-                        </li>
-                        <li className="menu-item">
-                            <a
-                                className="usa-nav-link"
-                                href="#/search"
-                                title="Search &amp; Download">
-                                <span>Search &amp; Download</span>
-                            </a>
-                        </li>
-                        <li className="menu-item">
-                            <SearchBar />
-                        </li>
-                    </ul>
+            </div>
+            <div className="mobile-menu">
+                <div className="mobile-button-wrapper">
+                    <button className={`usa-menu-btn ${homepageClass}`}>
```

Comment:
Not expanding the menu on click

Reply:
Menu does not exist, in design currently. Will be its own story

Reply:
Implementation will be in a future sprint